client/blocks/login/two-factor-authentication/verification-code-form.jsx

```diff
@@ -113,10 +128,11 @@ class VerificationCodeForm extends Component {
 				</p>
 
 				<hr />
-
-				<p>
-					<a href="#">{ translate( 'Send recovery code via text' ) }</a>
-				</p>
+				{ supportedAuthTypes.indexOf( 'sms' ) > -1 && (
```

Comment:
It's worth considering to have `isTwoFactorAuthTypeSupported( state, type )` selector instead of `getTwoFactorSupportedAuthTypes( state )`.